storybookjs/storybook
# Core: Fix `enum` args parsing from URL

Issue: N/A

If you have a component with an Typescript enum-type as input, it will be ignored by the args querystring, as it is rendered incompatible.

## What I did

Added 'enum' as a compatible type to be passed as args in URL

## How to test

Added spec for enum types in `args.test.ts`

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
